pkg/scheduler/nodeinfo/node_info.go

```diff
@@ -665,14 +665,20 @@ func (n *NodeInfo) FilterOutPods(pods []*v1.Pod) []*v1.Pod {
 			continue
 		}
 		// If pod is on the given node, add it to 'filtered' only if it is present in nodeInfo.
-		podKey, _ := GetPodKey(p)
+		var podKey string
```

Comment:
pre-declaring these variables doesn't do much for us here. How about something like this instead?
```go
podKey, err := GetPodKey(p)
if err != nil {
  klog.Errorf...
}
```